components/MediaPreview/MediaControlBar.tsx

```diff
@@ -140,6 +143,10 @@ function MediaControlBar({
   }, [setError]);
 
   useEffect(() => {
+    if (!router.isReady) {
+      return;
+    }
+
```

Comment:
The only component/side-effect that has to wait for the router to be ready, is the side-effect where we `getUserMedia` right after media preview mounts, and video is enabled. The reason is that we need the query parameter for simulcast before enabling the video.